Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.11
+
+RUN apk add --no-cache coreutils curl jq gfortran libc-dev cmake make python3 git
+
+WORKDIR /opt/test-runner/testlib
+RUN curl -R -O https://raw.githubusercontent.com/pclausen/fortran/main/testlib/CMakeLists.txt
+RUN curl -R -O https://raw.githubusercontent.com/pclausen/fortran/main/testlib/TesterMain.f90
```

Comment:
@pclausen The test runner runs without networking, which means that the testing library can't be downloaded on the fly when running the tests. The solution is to pre-download the testing library within the Dockerfile, which does have access to the network when being built.

Reply:
ahh, did not know that.